website/templates/emails/notify_base.mako

```diff
@@ -43,7 +43,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings">user settings</a>.</p>
```

Comment:
Yes and I don't think so, since its a base template I cannot be sure where the mako variables will be populated, so I could for my engagement mails, but it would throw a mako error for any other template using it.